src/checkpoint.rs

```diff
@@ -0,0 +1,11 @@
+//Return an array of satic ordered checkpoints
+fn get_checkpoint(block_height: i32, is_testnet: bool) -> (i32, &'static str) {
```

Comment:
```suggestion
fn get_checkpoint(block_height: i32, is_testnet: bool) -> Option<(i32, &'static str)> {
```
Change this to an option so we don't have to panic if there is no available checkpoint